test/AutoRest.TestServerLowLevel.Tests/paging.cs

```diff
@@ -157,12 +157,12 @@ await foreach (var page in pageableAsync.AsPages())
         public Task PagingMultiple_Continuation() => Test(async (endpoint) =>
         {
             var client = new PagingClient(Key, endpoint, null);
-            var pages = client.GetMultiplePages(null, null, null, new()).AsPages().ToArray();
+            var pages = client.GetMultiplePages(null, null, new()).AsPages().ToArray();
```

Comment:
What triggered the change in this file?

Reply:
Yes. in the paging service, it define `client-request-id` header, we need to remove it from method signature.

Reply:
It's defined here: https://github.com/Azure/autorest.testserver/blob/a949278ab15ec464bfa71da2a9e9c518be022b50/swagger/paging.json#L157-L162